docs/source/release/v3.13.0/framework.rst

```diff
@@ -39,6 +39,7 @@ Improved
 
 - :ref:`ConjoinXRuns <algm-ConjoinXRuns>` joins Dx errors if present
 - The algorithm :ref:`SortXAxis <algm-SortXAxis>` has a new input option that allows ascending (default) and descending sorting. Furthermore, Dx values will be considered if present. The documentation needed to be corrected.
+- :ref:`Live Data <algm-StartLiveData>` for events in PreserveEvents mode now produces workspaces that have reasonable bin boundaries.
```

Comment:
could you swap out the word "reasonable" for "that encompass the total x-range (TOF) for all events across all spectra" or something like that?